app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(10, 3)
```

Comment:
The assignment of `result` is unnecessary for this simple test. You can directly use the function call in the `assert` statement as per the checklist recommendation.